docs/integration_resources/resources/usd_tutorials/usd_tutorial_01.md

```diff
@@ -0,0 +1,68 @@
+## Creating Your First USD Stage
+
+This tutorial is focused on creating a simple USD stage containing a transform and a sphere using the operators from Shift_USD catalog.
+
+
+### Tutorial
+
+1. Open Shift application. A new window's instance of Shift should appear like the following one:
+
+<figure>
+    <img src="images/usd_tutorial_01/step_01_t01.png" alt="Shift UI.">
+    <figcaption><b>Figure 1</b>: Shift UI.</figcaption>
+</figure>
+
+>[!NOTE]
+> Check if the USD nodes appear in the Node List widget, it can be easily confirmed if in the catalog combobox appear `sUSD` or `sUSDTypes` catalogs. If no USD nodes appear, check again the [Shift_USD installation page](../usd.md#installation).
+
+2. Now, add a new **<tt>UsdStageNew</tt>** node to the board. Using the **Inspector** widget, set as the value for the **filepath** input plug a filepath pointing to a `.usda` file. For the current example, the filepath can be: `D:/USD_TUTORIALS/01/HelloWorld.usda`. The board should look like the following:
```

Comment:
Why are you using `<tt>`? This has been deprecated in html

Reply:
Oh, yes, I forgot it, you're right.
I have 2 proposals for replacing the `<tt>` tag:

### Using the `<code>` tag:
![Code_Tag](https://github.com/user-attachments/assets/607457cd-a8fb-4ebe-b2f8-7ebb4e8a14b3)

### Using bold-italic font:
![Bold_italic](https://github.com/user-attachments/assets/981d76df-942c-4118-bc7c-73aa20657691)

Please @Inbibo/shift , let me know which is the best way for use it when naming the node/operators, referring the name of the node to be searched in the Node List widget for filtering.

Reply:
Okay, I changed it for the tutorial 01 and has this appearance:
![image](https://github.com/user-attachments/assets/83f49ce8-eccf-4386-a838-79ca7f6012d4)

Let me know what you do think adding the reaction: 👍  or 👎